packages/web-components/fast-foundation/src/tabs/tabs.ts

```diff
@@ -162,7 +171,11 @@ export class Tabs extends FoundationElement {
     }
 
     private setTabs = (): void => {
-        const gridProperty: string = this.isHorizontal() ? "gridColumn" : "gridRow";
+        const gridHorizontalProperty: string = "gridColumn";
```

Comment:
Could these string values be good candidates for enum?

Reply:
This is a good point but considering only two values here, I don't anticipating that changing and these are the strings that were originally here I feel like it's best to keep it at strings for now. If there was a utility enum with all the possible values available there that would be awesome, another PR though I think.